test/metabase/lib/drill_thru/test_util/canned.cljc

```diff
@@ -164,6 +164,34 @@
      :aggregations    0
      :breakouts       0}
 
+    :test.query/products-native
+    {:query          (-> (lib/native-query metadata-provider "SELECT * FROM products")
+                         (assoc-in [:stages 0 :lib/stage-metadata]
+                                   {:columns (->> (meta/fields :products)
+                                                  (map #(meta/field-metadata :products %))
+                                                  (sort-by :position)
+                                                  (map #(select-keys % [:lib/type :name :display-name :field-ref
+                                                                        :base-type :effective-type :semantic-type])))
+                                    #_[{:lib/type :metadata/column, :name "ID",         :base-type :type/Integer}
+                                              {:lib/type :metadata/column, :name "EAN",        :base-type :type/Text}
+                                              {:lib/type :metadata/column, :name "TITLE",      :base-type :type/Text}
+                                              {:lib/type :metadata/column, :name "CATEGORY",   :base-type :type/Text}
+                                              {:lib/type :metadata/column, :name "VENDOR",     :base-type :type/Text}
+                                              {:lib/type :metadata/column, :name "PRICE",      :base-type :type/Float}
+                                              {:lib/type :metadata/column, :name "RATING",     :base-type :type/Float}
+                                              {:lib/type :metadata/column, :name "CREATED_AT", :base-type :type/DateTime}]}))
```

Comment:
Is this intentional?

Reply:
No that was left over from a REPL session to figure out the right shape for the `:columns`.